components/ww3/cime_config/config_compsets.xml

```diff
@@ -11,37 +11,37 @@
 
   <compset>
     <alias>VWW3-IAF</alias>
-    <lname>2000_DATM%IAF_SLND_SICE_SOCN_SROF_SGLC_WW3</lname>
+    <lname>2000_DATM%IAF_SLND_SICE_SOCN_SROF_SGLC_WW3%sp50x36</lname>
   </compset>
 
   <compset>
     <alias>VWW3-CFSv2</alias>
-    <lname>2000_DATM%CFSv2_SLND_SICE_SOCN_SROF_SGLC_WW3</lname>
+    <lname>2000_DATM%CFSv2_SLND_SICE_SOCN_SROF_SGLC_WW3%sp50x36</lname>
   </compset>
 
   <compset>
     <alias>VWW3-CFSR</alias>
-    <lname>2000_DATM%CFSR_SLND_SICE_SOCN_SROF_SGLC_WW3</lname>
+    <lname>2000_DATM%CFSR_SLND_SICE_SOCN_SROF_SGLC_WW3%sp50x36</lname>
   </compset>
 
   <compset>
     <alias>GMPAS-IAF-WW3</alias>
-    <lname>2000_DATM%IAF_SLND_MPASSI_MPASO%DATMFORCED_DROF%IAF_SGLC_WW3</lname>
+    <lname>2000_DATM%IAF_SLND_MPASSI_MPASO%DATMFORCED_DROF%IAF_SGLC_WW3%sp50x36</lname>
   </compset>
 
   <compset>
     <alias>GMPAS-JRA1p4-WW3</alias>
-    <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA-1p4-2018_SGLC_WW3</lname>
+    <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA-1p4-2018_SGLC_WW3%sp50x36</lname>
   </compset>
 
   <compset>
     <alias>WCYCLSSP585-WW3</alias>
-    <lname>SSP585SOI_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_WW3</lname>
+    <lname>SSP585SOI_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_WW3%sp50x36</lname>
   </compset>
 
   <compset>
     <alias>WCYCL1850-WW3</alias>
-    <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_WW3</lname>
+    <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_WW3%sp50x36</lname>
```

Comment:
Just wondering if you were you planning on adding compsets for the reduced spectral grids?

Reply:
My thinking is for the default setting you can just add WW3 to the compset alias (no need to define the sepctral res) the default will assume the spectral res for 36x36.  However, since we haven't yet uploaded those unresolved obstacles files yet to the ww3 input file directory this defualt option when using the 'alias compset' still uses 50x36 (as seen here by the long name definition). 

Ultimatly, If you want to use a non-defulat WW# spectra you would have to define you case with the long name and include either "*_WW3%sp50x36_*" or "*_WW3%sp25x36_*"  

I can add aliases for all the options but I figured that since most users will only want the default spectral res, its easiest to only have one compset alias.

Reply:
Great, that makes sense.